src/panels/config/integrations/integration-panels/voice_assistant/assist/assist-pipeline-debug.ts

```diff
@@ -0,0 +1,105 @@
+import "@material/mwc-button";
+import "@material/mwc-textfield";
+import { css, html, LitElement, TemplateResult } from "lit";
+import { customElement, property, query, state } from "lit/decorators";
+import "../../../../../../components/ha-card";
+import {
+  PipelineRun,
+  runPipelineFromText,
+} from "../../../../../../data/voice_assistant";
+import "../../../../../../layouts/hass-subpage";
+import { SubscribeMixin } from "../../../../../../mixins/subscribe-mixin";
+import { haStyle } from "../../../../../../resources/styles";
+import { HomeAssistant } from "../../../../../../types";
+
+@customElement("assist-pipeline-debug")
+export class AssistPipelineDebug extends SubscribeMixin(LitElement) {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ type: Boolean }) public narrow!: boolean;
+
+  @query(".run-pipeline-card mwc-textfield", true)
```

Comment:
```suggestion
  @query("#run-input", true)
```
Use `id` instead of class.